governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
+## Op1st Community Governance 1.0 (MVG)
+
+A healthy open source project is one that demonstrates open practices, uses open infrastructure, and cultivates an open culture with the goal of becoming more sustainable.
+
+### Principles
+As a new cloud-native community growing up at the footsteps of massively scaled and successful projects such as Kubernetes and OpenInfra, we want to be careful to learn from those who came before us, while forging our own path into the challenges we've chosen to undertake.
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
```

Comment:
Lets link the words [open source license] to the OSI approved license list: https://opensource.org/licenses

Reply:
One more factor. People will ask (as they have already asked me) why the AGPL was not mandated for some/all of this content. Do we have an ADR on this? Does that decision need to be recorded? Otherwise, this will be an obvious FAQ, and likely sore point.

Reply:
> Lets link the words [open source license] to the OSI approved license list: https://opensource.org/licenses

Surprisingly, this is why I didn't want to link to a specific list of licenses. We should have a community discussion around this. For example, I recommend using the [Fedora licensing page](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing) with some contextual explanations, such as how to use the list. Some people will want a specific subset of licenses, and those arguments should be heard.

If we start by linking to the full list of licenses, even if it's just to refer to what we mean by "open source licenses", we have defined a list for people. Then if we want to do any of the above suggestions as a community, we are in the position of taking something away from people, after just giving it to them.

So the reasoning here is to leave it undefined but clear enough, then clarify that in Governance 1.1.

(We'll come back to this PR to mine it for 1.1 specifics, which I'll mark with a :heart: for now.)

How does that sound to you?

Reply:
That works for me. As long as we describe this thought process, otherwise the question will arise in people's minds.

Reply:
Resolving this to make it clear there is no further action needed for Gov 1.0. We'll come back to this to seed the discussions for Gov 1.1.